crates/ethereum/node/src/node.rs

```diff
@@ -126,6 +132,61 @@ where
     }
 }
 
+/// Ethereum storage that implements [`ChainStorageReader`].
+#[derive(Debug, Default)]
+pub struct EthStorage;
```

Comment:
I assume what we want here instead is a generic variant because for both eth and op this will look the same.